lang/cs/Org.Apache.REEF.IO/FileSystem/IFileSystem.cs

```diff
@@ -106,9 +106,10 @@ public interface IFileSystem
         IEnumerable<Uri> GetChildren(Uri directoryUri);
 
         /// <summary>
-        /// Returns Url prefix 
+        /// Create Uri from a given file name
```

Comment:
Assumed path can be relative, or contains prefix already. We need to be able to handle both cases. 
I have added the check for null, and verification for the prefix. 
If the path cannot be used to create Uri, in both relative and full cases, then let the caller to handle the exception.